roles/famedly_sync/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: "Create volume path"
+  file:
+    path: "{{ item }}"
+    state: "directory"
+    mode: "0700"
```

Comment:
I usually prefer `0755` for the base directory so listing is allowed for everyone.

Reply:
Then, everyone would be able to read the "cache" (user database) that is created by famedly-sync on the first run.